test/azure.cs

```diff
@@ -83,8 +82,16 @@ public async Task small_file_azure()
             var prefix = PersistContainer.Uri + "/a/B2EA9F7FCEA831A4A63B213F41A8855B";
             Assert.Equal(prefix, url.Substring(0, prefix.Length));
 
-            var suffix = "&sp=r&rsct=application%2Foctet-stream&rscd=attachment%3Bfilename%3D\"test.bin\"";
-            Assert.Equal(suffix, url.Substring(url.Length - suffix.Length));
+            // HINT: suffix is no longer constructed in the same order for the download URL.
+            //var suffix = "&sp=r&rsct=application%2Foctet-stream&rscd=attachment%3Bfilename%3D\"test.bin\"";
+            //Assert.Equal(suffix, url.Substring(url.Length - suffix.Length));
+            
+            // Alternative way that doesn't asume specific order for query strings in the URL.
+            Assert.Contains("&sp=r", url);
+            Assert.Contains("&rsct=application%2Foctet-stream", url);
+            Assert.Contains("&rscd=attachment%3Bfilename%3D\"test.bin\"", url);
+            Assert.Contains("test.bin", url);
+            Assert.Contains("sig=", url);
```

Comment:
I understand that the suffix is no longer constructed in the same order, however if the construction is deterministic, we would prefer an equality assertion instead of multiple "contains" assertions: 

 - Equality detects any changes in the suffix (including the appearance of extraneous characters)
 - The expected shape of the suffix is much more obvious with equality than with contains